gap/elements/bipart.gi

```diff
@@ -93,6 +93,14 @@ InstallGlobalFunction(Bipartition,
 function(classes)
   local n, copy, i, j;
 
+  n := Sum(classes, Length) / 2;
+
+  if n >= 2 ^ 31 then
+    ErrorNoReturn("Semigroups: Bipartition: usage,\n",
+                  "the argument <classes> must be a list of lists whose union",
+                  "has length at most 2 ^ 31 - 1,");
```

Comment:
I that think the function takes a list of lists as its argument and the degree is half the length of the union of the lists.